config/plugins.ts

```diff
@@ -1,22 +1,22 @@
-module.exports = ({ env }) => ({
+export default () => ({
```

Comment:
**question(non-blocking):** It might be useful for folks looking at this file in the future to know they have access to the `env` parameter.  Is there an easy way to keep it?

Reply:
Yeah, that's fine, but we should prefix it with an `_` to show that it's unused